packages/mdc-ripple/foundation.js

```diff
@@ -325,27 +325,27 @@ class MDCRippleFoundation extends MDCFoundation {
 
     // Avoid reacting to follow-on events fired by touch device after an already-processed user interaction
     const previousActivationEvent = this.previousActivationEvent_;
-    const isSameInteraction = previousActivationEvent && e && previousActivationEvent.type !== e.type;
+    const isSameInteraction = previousActivationEvent && e !== undefined && previousActivationEvent.type !== e.type;
```

Comment:
I see at least 5 places where `!== undefined` was added... but isn't it unnecessary when undefined is the only acceptable falsy value?

Reply:
This case is to ensure the event object is present. I’d lean towards strict type checking.